C#/v1/NoarkWsClientSample/NoarkWsClientSample/IntegrationSample.cs

```diff
@@ -0,0 +1,355 @@
+                Console.WriteLine($"Could not find an document type code value '{documentType}'!");
+                return;
+            }
+
+            SubmitData(series,
+                primaryClass,
+                secondaryClass,
+                new AdministrativEnhet(administrativeUnitCodeValue.Code),
+                new Skjerming(screeningCodeValue.Code),
+                new Dokumenttype(documentTypeCodeValue.Code),
+                caseFileTitle,
+                caseFileExternalId,
+                caseResponsibleName,
+                caseResponsibleId,
+                registryEntryTitle,
+                registryEntryExternalId);
+        }
+
+        private void SubmitData(
+            Arkivdel series,
+            Klasse primaryClass,
+            Klasse secondaryClass,
+            AdministrativEnhet administrativeUnit,
+            Skjerming screeningCode,
+            Dokumenttype documentType,
+            string caseFileTitle,
+            string caseFileExternalId,
+            string caseResponsibleName,
+            string caseResponsibleId,
+            string registryEntryTitle,
+            string registryEntryExternalId)
+        {
+            #region Case file
+
+            Saksmappe caseFile = new Saksmappe(caseFileTitle, administrativeUnit)
+            {
+                Saksansvarlig = caseResponsibleName,
+                SaksansvarligBrukerIdent = caseResponsibleId
+            };
+
+            EksternId caseFileExternalIdObj = new EksternId(EXTERNAL_SYSTEM, caseFileExternalId);
+
+            #endregion Case file
+
+            #region Registry entry
+
+            Journalpost registryEntry = new Journalpost(registryEntryTitle, Journalposttype.UTGAAENDE_DOKUMENT)
+            {
+                Skjerming = screeningCode
+            };
+
+            registryEntry.VirksomhetsspesifikkeMetadata.AddBsmFieldValues("gr-1", "f-string", "value 1");
+
+            EksternId registryEntryExternalIdObj = new EksternId(EXTERNAL_SYSTEM, registryEntryExternalId);
+
+            Korrespondansepart correspondenceParty =
+                new Korrespondansepart(Korrespondanseparttype.AVSENDER, "John Smith");
+
+            #endregion Registry entry
+
+            #region Documents
+
+            //Upload two files
+
+            Dokumentfil mainFile = UploadDocument(this.testFile1);
+            Dokumentfil attachmentFile = UploadDocument(this.testFile2);
+
+            //Link the first document description to the registry entry as main document (HOVEDDOKUMENT).
+            //Subsequent document descriptions will be linked as attachments (VEDLEGG).
+
+            Dokument mainDocumentDescription =
+                new Dokument("Main Document", TilknyttetRegistreringSom.HOVEDDOKUMENT)
+                {
+                    Dokumenttype = documentType,
+                };
+
+            Dokumentversjon mainDocumentVersion =
+                new Dokumentversjon(Variantformat.PRODUKSJONSFORMAT, ".pdf", mainFile);
```

Comment:
I would rather have ARKIVFORMAT here because that would be the most common scenario.
```suggestion
                new Dokumentversjon(Variantformat.ARKIVFORMAT, ".pdf", mainFile);
```

Reply:
Changed it.